src/include/migraphx/schedule.hpp

```diff
@@ -41,6 +41,7 @@ struct MIGRAPHX_EXPORT schedule
 {
     schedule_model model{};
     bool enable = true;
+    bool weight_streaming = false;
```

Comment:
The schedule shouldn't be aware directly of aware of weight_streaming. It doesnt make sense on non-async targets which wont have multiple streams. Instead, it we want to have some operators to be scheduled on specific stream then we should update the stream_model to support that.

Reply:
Ended up removing any streaming implementation in the scheduler for the time being given that it doesn't currently improve performance.